third_party/github.com/bazelbuild/rules_typescript/devserver/runfiles/runfiles.go

```diff
@@ -1,10 +1,34 @@
 // Package runfiles that provides utility helpers for resolving Bazel runfiles within Go.
 package runfiles
 
-import "github.com/bazelbuild/rules_go/go/tools/bazel"
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/bazelbuild/rules_go/go/tools/bazel"
+)
 
 // Runfile returns the base directory to the bazel runfiles
 func Runfile(_base string, manifestPath string) (string, error) {
-	return bazel.Runfile(manifestPath)
-}
+	// If the path is a absolute path then resolve it directly before adapting lookup path
+	// The below resolution logic is copied from the bazel.Runfile function
+	if _, err := os.Stat(manifestPath); err == nil {
+		return filepath.Abs(manifestPath)
+	}
+
+	// It seems like any npm dependency is referenced like 'cloud/../npm/node_modules/@angular/core/bundles/core.umd.js' and 'cloud/../npm/@bazel/typescript/third_party/npm/requirejs/require.js'
+	// This is true for both script dependencies and scriptsManifest
+	// If they are corrected to 'node_modules/@angular/core/bundles/core.umd.js' and '@bazel/typescript/third_party/npm/requirejs/require.js' the following if condition can be removed
+	if i := strings.Index(manifestPath, "/../npm"); i > 0 {
```

Comment:
I don't think this is right in general, users can name this workspace whatever they like. "npm" is just the most common name

Reply:
Maybe you need to use `ListRunfiles` like Jay suggested back here: https://github.com/bazelbuild/rules_go/pull/2076#issuecomment-520888200

Reply:
If using `ListRunfiles` I can also make it case insensitive. I don't believe web urls should not be case sensitive.